### ISSUE TITLE
Support TinyMCE Editor [proposed enhancement]

### ISSUE THREAD

**Issue by kostasx** [OleVik/grav-plugin-readability#1]
When Readability plugin is used alongside the [TinyMCE Editor plugin](https://github.com/newbthenewbd/grav-plugin-tinymce-editor), the functionality is no longer there as a JavaScript error is raised.

One proposed way to fix this issue and support both the default Grav Markdown editor as well as the TinyMCE editor is to add a conditional check on the [admin.js](https://github.com/OleVik/grav-plugin-readability/blob/master/js/admin.js#L84) file in order for the Readability scan to work on both content areas. One way to do that (already implemented on a client's site) is to add the following lines before the **renderAnalysis()** function is called:

```
const markdownEditor = document.querySelector('textarea[name="data[content]"]');
// Check if TinyMCE is activated and grab the value from the active editor or just get the value from the default markdown textarea:
const currentContent = typeof tinyMCE !== "undefined" ? tinyMCE.activeEditor.getContent({ format: 'text' }) : markdownEditor.value;
            
renderAnalysis(

    currentContent,
    // ORIGINAL CODE:
    // document.querySelector('textarea[name="data[content]"]').value,

```

This is just a suggestion. In case I find some free time, I will be doing a PR on this.

Thank you for the plugin! @OleVik

**Comment by OleVik** [comment]
That looks like an elegant and easy solution to the problem. If you would like time to submit a PR, I'm happy to wait for that.

**Comment by OleVik** [comment]
Implemented in [v1.1.0](https://github.com/OleVik/grav-plugin-readability/releases/tag/v1.1.0).
